### PR TITLE
Fixed post moderation flaky test

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Flaky test fix

## Description

Fixes a flaky E2E test for post moderation that is currently on the main branch.

## QA Instructions, Screenshots, Recordings

N/A Fixing a flaky test only.

### UI accessibility concerns?

N/A Fixing a flaky test only, but it does ensure that an `aria-expanded` attribute is present and set to true when expanded.

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js [hunk 1]
```diff
@@ -48,9 +48,16 @@ describe('Moderation Tools for Posts', () => {
...
         cy.getIframeBody('[title="Moderation panel actions"]').within(() => {
-          cy.findByRole('button', { name: 'Open adjust tags section' }).click({
-            force: true,
-          });
+          cy.findByRole('button', { name: 'Open adjust tags section' })
+            .as('adjustTagsButton')
+            .click({
+              force: true,
+            });
+          cy.get('@adjustTagsButton').should(
```
Comment (nickytonline) [inline cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js:56]: Checking that `aria-expanded` is set to `"true"` means the panel should be open making the tag buttons discoverable to the test.

Comment (djuber) [inline cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js:56]: `should` here waits until it's true (or fails). This makes sense.

**Review by djuber — APPROVED** [review]

**Review by djuber — APPROVED** [review]

**Review by juliannatetreault — APPROVED** [review]
The Travis failure appears unrelated, so ✔️

**Comment by nickytonline** [comment]
> The Travis failure appears unrelated, so ✔️

I restarted that E2E node on Travis. It could potentially be another flaky test. I'm going to take a peek at it.

```
      (Attempt 3 of 4) should remove display ad if confirmation text matches
2022-02-08 18:45:15 +0000 Rack app ("DELETE /admin/customization/display_ads/1" - (127.0.0.1)): #<ActiveRecord::QueryCanceled: PG::QueryCanceled: ERROR:  canceling statement due to statement timeout
CONTEXT:  while deleting tuple (0,1) in relation "display_ads"
>
      1) should remove display ad if confirmation text matches
      ✓ generates error message when remove action fails (3594ms)
  3 passing (50s)
  1 failing
  1) Display Ads
       delete a display ad
         should remove display ad if confirmation text matches:
     TestingLibraryElementError: Timed out retrying after 4000ms: Unable to find an element by: [data-testid="snackbar"]
      at Object.getElementError (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:7986:17)
      at eval (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:8972:25)
      at eval (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:8944:24)
      at eval (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:8997:25)
      at baseCommandImpl (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:7431:16)
      at commandImpl (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:7434:40)
      at getValue (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:7458:23)
      at resolveValue (http://127.0.0.1:41829/__cypress/tests?p=cypress/support/index.js:7498:35)
```

**Pull request merged**